src/main/java/seedu/address/model/transaction/Timestamp.java

```diff
@@ -4,15 +4,27 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
 
-
 /**
  * Represents a Transaction's timestamp in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidTimestamp(String)}
  */
 public class Timestamp {
-    public static final String MESSAGE_CONSTRAINTS = "Timestamp must be in a valid ISO datetime format";
+    public static final String MESSAGE_CONSTRAINTS = "Date must be in DD/MM/YYYY format "
+            + "and time must be in HH:MM format; date should come before time if both are provided";
```

Comment:
When you say `if both are provived`, do you mean that users can choose to provide only time or only date?

Reply:
Also this behaviour needs to be documented in the UG